k8s-daemonset/k8s/linkerd-grpc.yml

```diff
@@ -28,10 +28,10 @@ data:
       label: outgoing
       experimental: true
       dtab: |
-        /srv                      => /#/io.l5d.k8s/default/grpc;
-        /svc/helloworld.Hello     => /srv/hello;
-        /svc/helloworld.World     => /srv/world;
-        /srv/world                => /srv/world-v1;
+        /srv        => /#/io.l5d.k8s/default/grpc;
+        /grpc       => /srv;
+        /grpc/World => /srv/world-v1;
+        /svc        => /$/io.buoyant.http.domainToPathPfx/grpc;
       identifier:
         kind: io.l5d.headerPath
```

Comment:
TIOLI: I think setting `segments` to 1 in the identifiers will clean up some unsightly residuals since we don't care about the gRPC method name for routing.